examples/sweeper/src/scripts/index.ts

```diff
@@ -0,0 +1,152 @@
+import * as path from "path";
+import * as dotenv from "dotenv";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+import { ethers } from "ethers";
+import type { Token } from "@uniswap/sdk-core";
+import { toReadableAmount } from "../utils";
+import { getProvider, getTurnkeySigner } from "../provider";
+import {
+  ERC20_ABI,
+  UNI_TOKEN_GOERLI,
+  USDC_TOKEN_GOERLI,
+  WETH_TOKEN_GOERLI,
+} from "../utils";
+
+const prompts = require("prompts");
```

Comment:
Need `prompts` in package.json, and `@types/prompts` as a devDependency

Reply:
Then you can convert this to an `import`

Reply:
💯 thanks